staging/src/k8s.io/legacy-cloud-providers/aws/tags.go

```diff
@@ -152,6 +152,15 @@ func (t *awsTagging) hasClusterTag(tags []*ec2.Tag) bool {
 	return false
 }
 
+func (t *awsTagging) hasNoClusterPrefixTag(tags []*ec2.Tag) bool {
+	for _, tag := range tags {
+		if strings.HasPrefix(aws.StringValue(tag.Key), TagNameKubernetesClusterPrefix) {
```

Comment:
Just out of curiosity, what's the reason for excluding the legacy cluster tag here?

Reply:
There is no major reason for excluding the legacy cluster tag. It is fairly old, EKS doesn't use the tag and I was thinking more of dropping the legacy burden. Let me know if we need to support the legacy tag. I intend to backport this change to 1.19 as well.

Reply:
@justinsb or @nckturner probably have more context on this. I would be interested to know if there are people still using legacy cluster role tag and if we should consider it here. Not blocking though